src/main/resources/add-ons-to-index.json

```diff
@@ -6750,6 +6750,177 @@
         }
       ]
     },
+    {
+      "uid": "refapp_2.10",
+      "name": "RefApp 2.10",
+      "description": "Included in Reference Application 2.10",
+      "addOns": [
+        {
+          "uid": "org.openmrs.module.addresshierarchy",
+          "version": "2.11.0"
+        },
+        {
+          "uid": "org.openmrs.module.admin-ui-module",
+          "version": "1.3.0"
+        },
+        {
+          "uid": "org.openmrs.module.allergy-ui-module",
+          "version": "1.8.2"
+        },
+        {
+          "uid": "org.openmrs.module.appframework",
+          "version": "2.14.0"
+        },
+        {
+          "uid": "org.openmrs.module.appointmentscheduling",
+          "version": "1.12.0"
+        },
+        {
+          "uid": "org.openmrs.module.appointment-scheduling-ui-module",
+          "version": "1.9.0"
+        },
+        {
+          "uid": "org.openmrs.module.atlas",
+          "version": "2.2"
+        },
+        {
+          "uid": "org.openmrs.module.attachments",
+          "version": "2.2.0"
+        },
+        {
+          "uid": "org.openmrs.module.calculation",
+          "version": "1.2"
+        },
+        {
+          "uid": "org.openmrs.module.core-apps-module",
+          "version": "1.28.0"
+        },
+        {
+          "uid": "org.openmrs.module.data-exchange-module",
+          "version": "1.3.3"
+        },
+        {
+          "uid": "org.openmrs.module.emrapi",
+          "version": "1.28.0"
+        },
+        {
+          "uid": "org.openmrs.module.event",
+          "version": "2.8.0"
+        },
+        {
+          "uid": "org.openmrs.module.openmrs-fhir-module",
+          "version": "1.19.0"
+        },
+        {
+          "uid": "org.openmrs.module.form-entry-app-module",
+          "version": "1.4.2"
+        },
+        {
+          "uid": "org.openmrs.module.htmlformentry",
+          "version": "3.10.0"
+        },
+        {
+          "uid": "org.openmrs.module.htmlformentryui",
+          "version": "1.10.0"
+        },
+        {
+          "uid": "org.openmrs.module.htmlwidgets",
+          "version": "1.10.0"
+        },
+        {
+          "uid": "org.openmrs.module.idgen",
+          "version": "4.5.0"
+        },
+        {
+          "uid": "org.openmrs.module.legacy-ui-module",
+          "version": "1.6.0"
+        },
+        {
+          "uid": "org.openmrs.module.metadatadeploy",
+          "version": "1.11.0"
+        },
+        {
+          "uid": "org.openmrs.module.metadatamapping",
+          "version": "1.3.4"
+        },
+        {
+          "uid": "org.openmrs.module.metadatasharing",
+          "version": "1.6.0"
+        },
+        {
+          "uid": "org.openmrs.module.namephonetics",
```

Comment:
Are you sure this module is included?

Reply:
Resolved, this was removed but maintained its version in distro